src/core/extended-language-client.ts

```diff
@@ -121,14 +88,28 @@ export interface DocumentIdentifier {
     uri: string;
 }
 
-export interface BallerinaAstOasChangeResponse {
-    oasAST?: string;
+export interface Range {
+    startLine: Position;
+    endLine: Position;
+}
+
+export interface Position {
```

Comment:
Why do we have two data models for `Position` and `PositionObject` with similar signature? Can't we rely on one? Maybe we can have a fix from LS side as well?

Reply:
We need two interfaces to differentiate io.ballerina.tools.text.LinePosition and org.eclipse.lsp4j.Position data stuctures. We cannot directly use vscode.Position, which has similar structure as org.eclipse.lsp4j.Position, because vscode.Position has additional methods.
Would it be better to change the names?

Reply:
If that's so, let's expose one of them as `LinePosition` that would be ideal.

Reply:
Updated with https://github.com/ballerina-platform/plugin-vscode/pull/371/commits/f549d2e50bb9a56c888863c3f3fb141a5bd24068